src/Fixtures/en/frisbeeDrills.en.js

```diff
@@ -5209,7 +5213,8 @@ const breakDrill = {
   author: 'Friselis',
   title: 'Cuts to the Break Side',
   image: 'https://zupimages.net/up/20/19/83fx.jpg',
-  description: 'Cutting from the open side to the break side. The drill is both for the thrower and the mark.',
+  description:
+    'Cutting from the open side to the break side. The drill is both for the thrower and the mark.\n\nIn game: Defenders seek to direct play by limiting the options for passing through a mark. If the attackers manage to break, they drastically increase the passing options.',
```

Comment:
"direct play" ça se dit ? on pourrait mettre "Defenders seek to limit the options..."